charts/seed-bootstrap/charts/nginx-ingress/templates/_helpers.tpl

```diff
@@ -0,0 +1,9 @@
+{{- define "nginx-ingress.config.data" -}}
+{{- range $k, $v := .Values.config }}
+{{ $k }}: {{ quote $v }}
+{{- end }}
+{{- end -}}
+
+{{- define "nginx-ingress.config.name" -}}
+nginx-ingress-controller-{{ include "nginx-ingress.config.data" . | sha256sum | trunc 8 }}
```

Comment:
I'm wondering if this works when there is no config specified for the ingress controller?

Reply:
Yeah, it works well, you can try locally by setting `config: {}` in the Helm chart.